demos/serverless/src/handlers.js

```diff
@@ -23,8 +24,8 @@ const chime = new AWS.Chime({ region: 'us-east-1' });
 chime.endpoint = new AWS.Endpoint(endpoint);
 
 const chimeSDKMeetings = new AWS.ChimeSDKMeetings({ region: currentRegion });
-if(endpoint != 'https://service.chime.aws.amazon.com'){
-  chimeSDKMeetings.endpoint = new AWS.Endpoint(endpoint);
+if(chimeSDKMeetingsEndpoint != 'https://service.chime.aws.amazon.com' && useChimeSDKMeetings === 'true'){
+  chimeSDKMeetings.endpoint = new AWS.Endpoint(chimeSDKMeetingsEndpoint);
```

Comment:
If this is not satisfied what is the `chimeSDKMeetings` endpoint? Is there a default value of `https://service.chime.aws.amazon.com` to it?

Reply:
Also, would it makes sense to just remove the `useChimeSDKMeetings` now that we are adding an explicit `chimeSDKMeetingsEndpoint` URL as new parameter?

Reply:
Yes, right, it will be pointing to Prod endpoint

Reply:
So `useChimeSDKMeetings` is basically true by default and says to use the `ChimeMeetingSDK` for regional endpoints and the `chimeSDKMeetingsEndpoint` actually specifies the regional `endpoint`. Be default, all deployments use the `ChimeMeetingSDK` regional endpoint leaving the canary test demos as we are deploying new demos to test the regional endpoints already.

Discussed offline but the clarification is ^.